Tests/SourceKitLSPTests/BuildSystemTests.swift

```diff
@@ -86,13 +89,15 @@ final class BuildSystemTests: XCTestCase {
   /// Whether clangd exists in the toolchain.
   var haveClangd: Bool = false
 
-  override func setUp() {
+  /// Must be called before every test execution.
+  ///
+  /// We can't override `XCTestCase.setup` because this method is async.
+  func setup() async {
```

Comment:
It would be ugly, but I suppose you could use a semaphore?

Reply:
That is ugly and very much an anti pattern. But then: It’s only test code and even if we get into a priority inversion here, it doesn’t really matter. I changed it to a semaphore.

Reply:
Yeah, that was my thinking too